Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM maven:3-jdk-8-onbuild-alpine
```

Comment:
When using the maven base image, will it build the project when building the image?